Tests/TestingApps/PaywallsTester/PaywallsTester/Data/ApplicationData.swift

```diff
@@ -34,6 +34,13 @@ final class ApplicationData {
         self.manager.signOut()
         self.authentication = .signedOut
     }
+    
+    var isSignedIn: Bool {
+        if case .signedIn = self.authentication {
+            return true
+        }
+        return false
+    }
```

Comment:
This is here so it's simple to get a bool of if the user is signed in or not, without worrying about the signed in enum's associated data.